scripts/tsplot

```diff
@@ -84,7 +82,7 @@ def parse_clargs():
     return P.parse_args(argsbis)
 
 def isstring(s):
-    return isinstance(s,str) or isinstance(s,unicode)
+    return isinstance(s,str) or isinstance(s,str)
```

Comment:
This should simplify to just
```Python
    return isinstance(s,str)
```
To the point where you can just call `isinstance(s,str)` directly inline.

Reply:
I used 2to3, I removed this and some other clutter in introduced.